spectral_cube/spectral_cube.py

```diff
@@ -2563,25 +2567,30 @@ def reproject(self, header, order='bilinear', use_memmap=False,
         shape_out = tuple([header['NAXIS{0}'.format(i + 1)] for i in
                            range(header['NAXIS'])][::-1])
 
+        if filled:
+            data = self.filled_data[:]
+        else:
+            data = self._data
```

Comment:
Is there a unit difference between `_data` and `filled_data`? Does it matter if the output is written to a memmap? This hasn't been a issue and is the default setting so maybe it doesn't matter.

Reply:
huh, should probbably be `unitless_filled_data` here....

Reply:
the units should be the same & inherited from the cube